### ISSUE TITLE
Code for the backdoor defense

### ISSUE THREAD

**Issue by fanghenshaometeor** [shaaaaron/DELETE#2]
Hi authors, a great work in machine unlearing without retain data!

I am interested in the backdoor defense part in your paper with results shown in Table S13. I have been tried hard to reproduce the results but failed. There are also very minor descriptions about the training settings in the paper. Could you please kindly release the code for this part? Many thanks!

**Comment by shaaaaron** [comment]
Thank you for your interest!

Our implementation is almost entirely adapted from [this repository](https://github.com/fmy266/Pytorch-Backdoor-Unlearning), and I recommend you make slight modifications directly based on their code.

However, please be aware of a few issues in the original code:

- Due to version differences, some parameters in trojanvision have been renamed or are no longer supported. From my recollection, the issue is related to one of one parameter in `main.py` between lines 12 and 22.
- The early stopping mechanism in the backdoor training introduces randomness into the experiments. We suggest increasing the number of epochs or adjusting the learning rate to mitigate this effect, and take the average performance of five runs under the same parameter settings.

I hope this is helpful to you!

**Comment by fanghenshaometeor** [comment]
Thanks for your kind reply and suggestions! I will try based on the shared repository.
